dot/network/discovery.go

```diff
@@ -35,13 +35,15 @@ var (
 	startDHTTimeout             = time.Second * 10
 	initialAdvertisementTimeout = time.Millisecond
 	tryAdvertiseTimeout         = time.Second * 30
-	connectToPeersTimeout       = time.Minute
+	connectToPeersTimeout       = time.Minute * 5
+	findPeersTimeout            = time.Minute
```

Comment:
nit: I prefer to put these configurable settings inside the struct, so `discovery` in this case.

Reply:
would you put them inside the struct then assign the value inside `newDiscovery`?

Reply:
Yea then you can set your desired timeout in the tests per `*discovery` rather than globally through the package.